mscore/shortcut.cpp

```diff
@@ -789,6 +789,17 @@ Shortcut Shortcut::_sc[] = {
          Qt::WindowShortcut,
          ShortcutFlags::A_CMD
          },
+      {
+         MsWidget::SCORE_TAB,
+         STATE_NORMAL | STATE_NOTE_ENTRY,
+         "pitch-up-diatonic-alterations",
+         QT_TRANSLATE_NOOP("action","Diatonic Up (Keep degree alterations"),
+         QT_TRANSLATE_NOOP("action","Diatonic pitch up (Keep degree alterations)"),
```

Comment:
Maybe better be more explicit? Like `QT_TRANSLATE_NOOP("action","Transpose selection diatonic pitch up (Keep degree alterations)"),`